.changeset/fix-tailwind-preset-cjs-interop.md

```
@@ -0,0 +1,13 @@
+---
+"@lynx-js/tailwind-preset": patch
+---
+
+Fix ESM/CJS interop for tailwindcss internal imports.
+
+The built output previously used named ESM imports from tailwindcss CJS modules whose exports cannot be detected by Node.js's `cjs-module-lexer` (due to the dynamic `_export()` helper pattern). This caused runtime errors like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)